hbase-protocol-shaded/src/main/protobuf/server/master/RegionServerStatus.proto

```diff
@@ -230,4 +242,8 @@ service RegionServerStatusService {
   /** Get some live region servers to be used as seed for bootstrap nodes */
   rpc GetLiveRegionServers(GetLiveRegionServersRequest)
     returns(GetLiveRegionServersResponse);
+
+  /** Report BrokenStoreFileCleaner chore metrics to master */
+  rpc ReportBrokenStoreFileCleanerUsage(BrokenStoreFileCleanerUsageRequest)
```

Comment:
Is it worth a separated method? How do we report the metrics for other background tasks on region server to master?

Reply:
Usually we do not report metrics. I do this here for 2 reasons:
- Convenience. Aggregating the rs metrics is possible and usually done by queries from the db, storing aggregated metrics here is for ease of use.
- Preparing things for the next PR where I'll add logic to store the reported metrics in-memory and extend the REST service to allow checking these values without having to access the metrics db.